### PR TITLE
(DOCSP-15892): Add --verbose and --quiet options

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
JIRA: [DOCSP-15892](https://jira.mongodb.org/browse/DOCSP-15892)

Staged: [mongosh options](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-15892/reference/options/)

### CODE REVIEW

**Review by addaleax — APPROVED** [review]

File: source/reference/options.txt [hunk 1]
```diff
@@ -393,12 +393,25 @@ Authentication Options
    :option:`--password <--password>` option as the last option and leave out the
    argument.
 
+.. option:: --quiet
+
+   Skip all messages during startup (such as welcome messages and startup
```
Comment (jmd-mongo) [inline source/reference/options.txt:398]: nit (non-blocking):

I took a look at the other examples and many (though not all) seem to be in the simple present (e.g. "Increases", "Specifies", "Bypasses"). Perhaps it's worthwhile to update the tense here ("Skip" -> "Skips", "go" -> "goes") for consistency. Just a thought!

File: source/reference/options.txt [hunk 2]
```diff
@@ -393,12 +393,25 @@ Authentication Options
...
    Specifies a username with which to authenticate to a MongoDB database
    that uses authentication. Use in conjunction with the
    :option:`--password <--password>` and
-   :option:`--authenticationDatabase <--authenticationDatabase>` options.
+   :option:`--authenticationDatabase <--authenticationDatabase>`
+   options.
+
+.. option:: --verbose
+
+   Increases the verbosity of the output of the shell during the connection
```
Comment (jmd-mongo) [inline source/reference/options.txt:411]: suggestion (non-blocking):

Consider "Increases the verbosity of the shell output..." May flow a bit better.

**Review by jmd-mongo — APPROVED** [review]
LGTM mod nits! 

Thanks!
joe

**Comment by jeff-allen-mongo** [comment]
Thanks @jmd-mongo! Incorporated your comments and will merge.

**Pull request merged**